audio/mods/maxtrax.cpp

```diff
@@ -996,7 +996,6 @@ bool MaxTrax::load(Common::SeekableReadStream &musicData, bool loadScores, bool
 	return false;
 }
 
-#if !defined(NDEBUG) && 0
```

Comment:
This was always disabled: `&& 0`

Reply:
Yeah, that block of code exists for debug purposes. I've converted it to use a `debug()` for output, with a high level, like the rest of our code